dask_ml/model_selection/_incremental.py

```diff
@@ -118,7 +122,14 @@ def _fit(
     fit_params=None,
     scorer=None,
     random_state=None,
+    verbose=False,
 ):
+    prefix = ""
+    if isinstance(verbose, tuple):
```

Comment:
Is this standard in scikit-learn?

Reply:
I'm not sure. I did this to simplify the data flow and didn't have to add another parameter to a bunch of private functions. This `verbose` originates in `SuccessiveHalvingSearchCV` initialization.

Reply:
I've removed this. Instead, I set an attribute (`'_prefix'`) on `BaseIncrementalSearchCV` and pass that to `_incremental.fit` (with `getattr(self, "_prefix", "")`).